doc/webservices/extract_api.rst

```diff
@@ -0,0 +1,478 @@
+                        },
+                },
+                "id": string (hex),
+            }
+
+Response
+^^^^^^^^
+
+* ``jsonrpc``: A string specifying the version of the JSON-RPC protocol. It will be “2.0”.
+* ``id``: The identifier you set in the request body.
+* ``result``: 
+    * ``status_code``: code indicating the status of the request. ``status_code`` is 0 in case of success. Other ``status_code`` are detailed in the table below.
+    * ``status_msg``: A string giving verbose details about the request status.
+    * ``document_id``: Only present if the request is successful.
+
+=============  ==============================================================
+ status_code       status_msg
+=============  ==============================================================
+0              Success
+2              An error occured
+3              You don't have enough credit
+6              Unsupported image format
+9              Server is currently under maintenance. Please try again later.
+=============  ==============================================================
+
+.. rst-class:: setup doc-aside
+
+.. switcher::
+
+    .. code-block:: json
+
+        {
+            "jsonrpc": string,
+            "id": string,
+            "result": {
+                "status_code": int,
+                "status_msg": string,
+                "document_id": int,
+            }
+        }
+
+
+.. _webservices/extract_api/invoice_get_results:
+
+/iap/invoice_extract/get_results (POST)
+'''''''''''''''''''''''''''''''''''''''
+
+Description
+^^^^^^^^^^^
+Request the results of the documents ids obtained with the :ref:`/parse <webservices/extract_api/invoice_parse>` route. Can either return the results or a "request pending" message.
+
+Request Body
+^^^^^^^^^^^^
+
+* ``jsonrpc``: Same as for :ref:`/parse <webservices/extract_api/invoice_parse>`. **REQUIRED**
+* ``method``: Same as for :ref:`/parse <webservices/extract_api/invoice_parse>`. **REQUIRED**
+* ``params``:
+    * ``version``: Same as for :ref:`/parse <webservices/extract_api/invoice_parse>`. **REQUIRED**
+    * ``documents_ids``: list of ``document_id`` for which you want to get the current parsing status.
+* ``id``: Same as for :ref:`/parse <webservices/extract_api/invoice_parse>`. **REQUIRED**
+
+.. rst-class:: setup doc-aside
+
+.. switcher::
+
+    .. code-block:: json
+
+        {
+            "jsonrpc": string,
+            "method": string,
+            "params": {
+                "version": int,
+                "documents_ids": [int]
+            },
+            "id": string (hex),
+        }
+
+
+Response
+^^^^^^^^
+
+* ``jsonrpc``:  Same as for :ref:`/parse <webservices/extract_api/invoice_parse>`.
+* ``id``:  Same as for :ref:`/parse <webservices/extract_api/invoice_parse>`.
+* ``result``:
+    For each ``document_id``:
+
+    * ``status_code``: code indicating the status of the request. ``status_code`` is 0 in case of success. Other ``status_code`` are detailed in the table below.
```

Comment:
```suggestion
    * ``status_code``: The code indicating the status of the request. ``status_code`` is 0 in case of success. Other ``status_code`` are detailed in the table below.
```
And so forth for the next occurrences.